exhale/graph.py

```diff
@@ -966,6 +999,7 @@ def __init__(self):
         self.root_directory         = configs.containmentFolder
         self.root_file_name         = configs.rootFileName
         self.full_root_file_path    = os.path.join(self.root_directory, self.root_file_name)
+        self.page_hierarchy_file    = os.path.join(self.root_directory, "page_view_hierarchy.rst")
```

Comment:
In docs request for table update on `begin_root_api_document_layout` in `docs/reference/configs.rst` please explain this is the file.  I've seen people manually post-process these files, so it's nice to know what it is.

Reply:
Done in https://github.com/svenevs/exhale/pull/114/commits/c4b141cc1fc852df08d74019a5dcb8cc3e2aa8c3